src/librustc_passes/liveness.rs

```diff
@@ -647,9 +638,8 @@ impl RWUTable {
 
 #[derive(Copy, Clone)]
 struct Specials {
```

Comment:
Pre-existing, but if we're going  to be extending this code, can we add a doc-comment explaining what a "special" is.